src/wasm2obj.rs

```diff
@@ -86,10 +86,11 @@ fn read_wasm_file(path: PathBuf) -> Result<Vec<u8>, io::Error> {
 }
 
 fn main() {
+	let version = env!("CARGO_PKG_VERSION");
```

Comment:
Indentation is off here and in the other files.

Reply:
Sorry, tab messed the indentation. Fixed to use spaces.